core/src/main/scala/fs2/netty/Network.scala

```diff
@@ -85,7 +85,8 @@ final class Network[F[_]: Async] private (
             Sync[F] delay {
               val bootstrap = new ServerBootstrap
               bootstrap.group(parent, child)
-                .option(JChannelOption.AUTO_READ.asInstanceOf[JChannelOption[Any]], false)   // backpressure
+                // TODO: does netty override this? `ServerBootstrap.init` via `bind` override schedules a future to set to autoread to false
```

Comment:
Yeah for sure. I discovered perchance when digging through Netty source and saw a scheduled task for autoread. Thought that was weird, so made a note to investigate. We can just print Netty config during runtime. Best case, our setting is always accepted and this is false concern. Worse case, we have to schedule an autoread override at the right time so that it executes after Netty's 🙃 .

Reply:
That would be abjectly horrible. :-D Like complain to Norman on Twitter horrible.

Reply:
haha will test it out :)